### PR TITLE
doc: Add instance sizing doc

### PR DESCRIPTION
**Pull request by jsirianni**
### Proposed Change

Added initial prerequisite doc, with a focus on instance sizing. I decided to target the following:
- xsmall: 1-10 agents
- small: 10 - 100 agents
- medium: 100 - 2,000 agents

The recommended resources are based directly on the instance sizes I used during a week long test, which seemed more than reasonable for the number of agents being managed. All observed systems were well below being maxed out, allowing the user to push a bit further before running into stability issues.

I will have a follow up PR for larger deployments.

##### Checklist
- [x] Changes are tested
- [x] CI has passed

### CODE REVIEW

**Review by Dylan-M — APPROVED** [review]
This looks like a good start. I'm fine with iterating it to add details for larger deployments later.

**Review by ryancgoins — APPROVED** [review]
Looks great

File: docs/www/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| 100         | 2 MB/s                  | 400/s  |
+| 1,000       | 113 MB/s                | 5000/s |
+
+### Network Requirements
+
+BindPlane OP maintains network connections for the following:
+- Agent Management
+- Agent Throughput Measurements
+- CLI and Web Interfaces
+
```
Comment (craigleeit) [inline docs/www/prerequisites.md:51]: The default port used is: TCP:3001
TLS can be implemented. For more information on TLS, See [TLS configuration section](doc:configuration#tls)

**Review by craigleeit — APPROVED** [review]
Looks good, added a note about ports and TLS. Those items come up in every customer engagement.

**Review by philcook-oiq — APPROVED** [review]
Looks great, this will help immensely!

**Review by shazlehu — APPROVED** [review]

**Pull request merged**